### ISSUE TITLE
Input field announcement

### ISSUE THREAD

**Issue by GreeshmakRaj** [tinymce/tinymce#6837]
**What is the current behavior? Describe the bug**
While using JAWS, If user presses INS + F5 to view form fields, the form field does NOT have a name.

**Please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tiny.cloud](https://fiddle.tiny.cloud/) or similar.**
https://www.tiny.cloud/docs/demo/basic-example/#
![image](https://user-images.githubusercontent.com/53045245/118594436-1abe8300-b7c7-11eb-9e0c-64f31904a5ca.png)

**What is the expected behavior?**
The text field should have a default label instead of Unlabeled(n) edit (address of the page). 
For a suggestion, user should be given property similar to placeholder, so that a proper label can be added.

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**
TinyMCE version - 5.2.1. 

Configurations 

- macOS Mojave Version 10.14.6 with Chrome 87.0.4280.88 (Official Build) (x86_64)
- Windows 10 with Chrome 87.0.4280.88 (Official Build) (x86_64) and JAWS 2018.1805.33
